deploy.sh

```diff
@@ -45,6 +43,7 @@ ssh -i $STAGING_KEY $STAGING_USER@$STAGING_SERVER << EOF
   sudo -u postgres psql -c "CREATE DATABASE timelessdb_test;"
   echo "-- REPLACE: add scripts to cron"
   cd /app
+  python -m venv venv
```

Comment:
@ekondrashev see line 13, there we execute `pytest`, but only here after 30 lines of code we are creating env.
Looks strange to me. @emilianodellacasa is it ok that we run `pytest` so early?

Reply:
@vryazanov Actually it's extra, but main idea here is to check if all tests are green before deploying to staging server. In theory we do it when merging, but you'll never know